pkg/console/operator/sync_v400.go

```diff
@@ -236,7 +236,16 @@ func SyncSecret(co *consoleOperator, recorder events.Recorder, operatorConfig *o
 // therefore no additional error handling is needed here.
 func SyncConfigMap(co *consoleOperator, recorder events.Recorder, operatorConfig *operatorv1.Console, consoleConfig *configv1.Console, infrastructureConfig *configv1.Infrastructure, rt *routev1.Route) (*corev1.ConfigMap, bool, error) {
 	logrus.Printf("validating console configmap...")
-	cm, cmChanged, cmErr := resourceapply.ApplyConfigMap(co.configMapClient, recorder, configmapsub.DefaultConfigMap(operatorConfig, consoleConfig, infrastructureConfig, rt))
+
+	defaultConfigmap, didMerge, configMergeMessage, err := configmapsub.DefaultConfigMap(operatorConfig, consoleConfig, infrastructureConfig, rt)
+	if didMerge {
+		co.UpgradableWithUnsupportedConfigOverrides(operatorConfig, configMergeMessage)
```

Comment:
So it looks like we never set upgrade anywhere else. Does that mean that the message will be wrong when I disable the unsupported config?

Reply:
@spadgett you mean when the `operatorConfig.Spec.UnsupportedConfigOverrides.Raw` ? Cause we are setting the status, when that field is set

Reply:
Same, removing since using `NewUnsupportedConfigOverridesController()`